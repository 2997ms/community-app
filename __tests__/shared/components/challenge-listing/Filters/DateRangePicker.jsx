import React from 'react';
import Renderer from 'react-test-renderer/shallow';
import DateRangePicker from 'components/challenge-listing/Filters/DateRangePicker';

const requiredProps = {
  onDatesChange: () => {},
  onFocusChange: () => {},
  startDateId: 'startDate',
  endDateId: 'endDate',
};

test('Matches shallow shapshot', () => {
  const renderer = new Renderer();
  renderer.render((
    <DateRangePicker {...requiredProps} />
  ));
  expect(renderer.getRenderOutput()).not.toBeNull();
});
