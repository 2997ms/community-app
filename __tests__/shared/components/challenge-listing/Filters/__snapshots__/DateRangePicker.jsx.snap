// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches shallow shapshot 1`] = `
<DateRangePicker
  anchorDirection="right"
  disabled={false}
  displayFormat="MMM DD, YYYY"
  enableOutsideDays={false}
  endDate={undefined}
  endDateId="endDate"
  focusedInput={null}
  horizontalMargin={0}
  initialVisibleMonth={[Function]}
  isDayBlocked={[Function]}
  isDayHighlighted={[Function]}
  isOutsideRange={[Function]}
  keepOpenOnDateSelect={false}
  minimumNights={1}
  monthFormat="MMMM YYYY"
  navNext={null}
  navPrev={null}
  numberOfMonths={2}
  onDatesChange={[Function]}
  onFocusChange={[Function]}
  onNextMonthClick={[Function]}
  onPrevMonthClick={[Function]}
  orientation="horizontal"
  phrases={
    Object {
      "clearDates": "Clear Dates",
      "closeDatePicker": "Close",
    }
  }
  reopenPickerOnClearDates={false}
  required={false}
  showClearDates={false}
  startDate={undefined}
  startDateId="startDate"
  withFullScreenPortal={false}
  withPortal={false}
/>
`;
